### PR TITLE
Various minor XTREME fixes

### PR DESCRIPTION
**Pull request by zphang**
* Sorting the UDPos files to ensure fixed order.
* Fix TyDiQA/XQuAD/MLQA reading
* Added `_config` back to JSON file name for SQuAD (for internal consistency, we can change it for all tasks later)
* Updated PAWS-X data reading (I think they changed the file format...)

Validated by WIP XTREME Benchmark code

### CODE REVIEW

File: jiant/tasks/lib/pawsx.py [hunk 1]
```diff
@@ -96,13 +96,11 @@ def get_test_examples(self):
     def _create_examples(cls, lines, set_type):
         examples = []
         for (i, line) in enumerate(lines):
+            if i == 0:
+                continue
```
Comment (jeswan) [inline jiant/tasks/lib/pawsx.py:100]: Can you add a comment explaining this?

File: jiant/tasks/lib/tydiqa.py [hunk 2]
```diff
@@ -45,10 +45,16 @@ def __init__(
         self.language = language
 
     def get_train_examples(self):
-        raise NotImplementedError("TyDiQA does not have training examples")
+        if self.language == "en":
+            #return super().get_train_examples()
```
Comment (jeswan) [inline jiant/tasks/lib/tydiqa.py:49]:
```suggestion
            
```

**Comment by zphang** [comment]
Sorry, the PR ended up having scope creep. I'll update and ping you.

File: jiant/tasks/lib/pawsx.py (hunk 1 above)
Comment (zphang) [reply jiant/tasks/lib/pawsx.py:100]: Added.

Comment (jeswan) [reply jiant/tasks/lib/pawsx.py:100]: Mind adding it again?

Comment (zphang) [reply jiant/tasks/lib/pawsx.py:101]: Whoops, forgot to push it.

Comment (jeswan) [reply jiant/tasks/lib/pawsx.py:101]: Can you explain why it is necessary to skip the first line? Is it a header?

Comment (zphang) [reply jiant/tasks/lib/pawsx.py:101]: Done.

**Pull request merged**